ingestion_tools/README.md

```diff
@@ -47,4 +62,27 @@ poetry shell
 poetry install
 ```
 
-**Note:** If you have multiple python versions, look at the guide [here](https://python-poetry.org/docs/managing-environments/) to configure your poetry environment to use a compatible python version.
+**Note:** If you have multiple python versions, refer to [this guide](https://python-poetry.org/docs/managing-environments/) to configure Poetry to use the correct version.
+
+
+## Useful Commands
+
+There are make commands available to run the ingestor, tests for the ingestion, pushing the image to remote, etc. They require docker to run.
+
+### Setup Docker
+
+If you don't already have docker setup, follow the instructions provided [here](https://www.docker.com/get-started/).
+
+`docker compose` utility is needed to execute the make commands, as the local application requires multiple containers to run simultaneously. Learn more about docker compose [here](https://docs.docker.com/compose/). To install `docker compose` follow the instructions provided [here](https://docs.docker.com/compose/install/).
+
+### Make Commands
```

Comment:
Note that these commands have to be run from the root of the repo?

Reply:
Added that below the table. :D